test/ProjectTemplates/Microsoft.Extensions.AI.Templates.IntegrationTests/Snapshots/aichatweb.AzureOpenAI_Qdrant_Aspire.verified/aichatweb/aichatweb.Web/aichatweb.Web.csproj

```diff
@@ -11,11 +11,15 @@
     <PackageReference Include="Aspire.Azure.AI.OpenAI" Version="9.5.1-preview.1.25502.11" />
     <PackageReference Include="Microsoft.Extensions.AI.OpenAI" Version="10.0.0" />
     <PackageReference Include="Microsoft.Extensions.AI" Version="10.0.0" />
-    <PackageReference Include="Microsoft.SemanticKernel.Core" Version="1.66.0" />
-    <PackageReference Include="PdfPig" Version="0.1.11" />
+    <PackageReference Include="Microsoft.Extensions.DataIngestion" Version="10.0.0" />
```

Comment:
Could you please explain why we are using `10.0.0` version for MEDI here? We are going to ship preview1 now.